r/R/dplyr-mutate.R

```diff
@@ -24,7 +24,9 @@ mutate.arrow_dplyr_query <- function(.data,
                                      .before = NULL,
                                      .after = NULL) {
   call <- match.call()
-  exprs <- ensure_named_exprs(quos(...))
+
+  expression_list <- unfold_across(.data, quos(...))
```

Comment:
I know you've deferred this from filter and summarize, but is that necessary? Is it more involved than adding in this `unfold_across` in them? There's no other modifications to mutate() so it seems reasonable to guess that it would be that straightforward in the other functions too.

Reply:
It's not as straightforward as I need to delve into the internals of `do_arrow_summarize()` and swap some bits around.  Given this PR is not tiny and I'd be wanting to add a lot of tests for all 3, I'd like to keep them separate to try to keep things simpler for review.